### PR TITLE
refresh price and pool info

### LINKED ISSUE #155: price & shares do not update after user interaction without browser refresh
the pool still thinks I have the same pool share even after withdrawing.
(from discord user)

### PR DESCRIPTION
**Pull request by mihaisc**
closes #155

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,7 +75,9 @@
     "remove-markdown": "^0.3.0",
     "shortid": "^2.2.16",
     "slugify": "^1.4.5",
+    "styled-components": "^5.2.0",
```
Comment (kremalicious) [inline package.json:78]: do we need that? That's a big dependency

Comment (mihaisc) [reply package.json:78]: noo, forgot it

Comment (mihaisc) [reply package.json:78]: it's one of the weird dependencies required when i use react hooks locally

**Review by kremalicious — APPROVED** [review]
testing on preview with adding liquidity, works perfectly

File: src/components/organisms/AssetActions/Pool/index.tsx [hunk 2]
```diff
@@ -110,18 +112,31 @@ export default function Pool({ ddo }: { ddo: DDO }): ReactElement {
         // swapFee is tricky: to get 0.1% you need to convert from 0.001
         const swapFee = await ocean.pool.getSwapFee(price.address)
         setSwapFee(`${Number(swapFee) * 100}`)
+
+        console.log(
```
Comment (kremalicious) [inline src/components/organisms/AssetActions/Pool/index.tsx:116]: maybe put them behind `Logger` or remove

**Pull request merged**